frontend/src/slices/styleDataSlice.ts

```diff
@@ -23,12 +21,14 @@ export const validStyleFunctions = {
   }
 };
 
-const initialState = {
+export const initialState = {
```

Comment:
Everything looks good but why is `initialState` being exported?

Reply:
It is used for when users want to revert certain aspects back to initial. For example, if they want only the taskbar background color to be reverted back to the original color, they can use initial. It is used in line 34 of TerminalAppScreen.tsx.

Reply:
Interesting, alright.